src/components/atoms/RecurringTransactionOptions.tsx

```diff
@@ -1,14 +1,16 @@
-import React from 'react';
+import React, { Dispatch, SetStateAction } from 'react';
 
 import Input from './Input';
 import Select from './Select';
 
+export type intervalType = 'month' | 'week';
```

Comment:
Custom types should have a capital first character.

Reply:
fixed 😃 Good catch!